tests/basic/basic_test.go

```diff
@@ -191,9 +192,36 @@ var _ = Describe("{VolumeDriverAppDown}", func() {
 					Expect(appNodes).NotTo(BeEmpty())
 				})
 
+				var appVolumes []*volume.Volume
+				Step(fmt.Sprintf("get volumes for %s app", ctx.App.Key), func() {
+					appVolumes, err = Inst().S.GetVolumes(ctx)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(appVolumes).NotTo(BeEmpty())
+				})
+
+				// avoid dup
+				nodesThatCantBeDown := make(map[string]bool)
+				nodesToBeDown := make([]node.Node, 0)
+				Step(fmt.Sprintf("choose nodes to be down for %s app", ctx.App.Key), func() {
+					for _, vol := range appVolumes {
+						replicas, err := Inst().V.GetReplicaSetNodes(vol)
+						Expect(err).NotTo(HaveOccurred())
+						Expect(replicas).NotTo(BeEmpty())
+						// at least 1 node with replica needs to be up
+						nodesThatCantBeDown[replicas[0]] = true
+					}
+
+					for _, node := range node.GetWorkerNodes() {
+						if _, exists := nodesThatCantBeDown[node.Name]; !exists {
```

Comment:
3 node PX cluster. repl=3 volumes.

Above code will keep 1 PX node up and bring down other 2 nodes. This will bring PX cluster out of quorum and hence all volumes are out of quorum.